src/oom/oomd-util.c

```diff
@@ -281,27 +281,45 @@ int oomd_cgroup_kill(const char *path, bool recurse, bool dry_run) {
 
 typedef void (*dump_candidate_func)(const OomdCGroupContext *ctx, FILE *f, const char *prefix);
 
-static int dump_kill_candidates(OomdCGroupContext **sorted, int n, int dump_until, dump_candidate_func dump_func) {
+static int dump_kill_candidates(
+                OomdCGroupContext **sorted,
```

Comment:
btw, i'd spell this as `OomdCGroupContext *sorted[]`. It's the same thing of course, but emphasizes this is an array